src/fielddef.ts

```diff
@@ -133,12 +133,13 @@ export interface FieldDefBase<F> {
   timeUnit?: TimeUnit;
 
   /**
-   * A flag for binning a `quantitative` field, or [an object defining binning parameters](https://vega.github.io/vega-lite/docs/bin.html#params).
+   * A flag for binning a `quantitative` field, [an object defining binning parameters](https://vega.github.io/vega-lite/docs/bin.html#params), or `"binned"`.
    * If `true`, default [binning parameters](https://vega.github.io/vega-lite/docs/bin.html) will be applied.
+   * If set to `"binned"`, it implies that the data is binned prior to being imported to Vega-lite. The scale and axis will be formatted similar to binning in Vega-lite.
    *
    * __Default value:__ `false`
    */
-  bin?: boolean | BinParams;
+  bin?: boolean | BinParams | 'binned';
```

Comment:
Try if we can override PositionFieldDef like discussed on slack?

Reply:
Done